scipy/cluster/tests/test_hierarchy.py

```diff
@@ -930,6 +930,26 @@ def test_dendrogram_colors(self):
         # reset color palette (global list)
         set_link_color_palette(None)
 
+    @pytest.mark.parametrize("order", [
```

Comment:
Add a parameterization with a duplicated value? You mention it in the description of the function, so we should probably explicitly test it here. Also, there should be a test for the happy path, where everything works as it should, and for the edge case of having an empty tree and empty list of reordering (or similar appropriate edge case).

Reply:
If I understand You correctly, than line 936 is a duplicated value. Maybe I should change the comment in that line. The rest of the tests You mention is on the way!

Reply:
@ihowell done!